knative-operator/hack/update-deps.sh

```diff
@@ -13,6 +13,10 @@ go mod vendor
 # TODO: Remove this once we bump kubernetes versions.
 git apply "$ROOT_DIR/hack/manifestival.patch"
 
+# This patch is required to make sure the cert names read by the webhook setup is equal
+# to the names chosen by OLM. We can drop this once we bump to newer versions of c-r.
+git apply "$ROOT_DIR/hack/certnames.patch"
+
```

Comment:
So, that patch existed previously but was not applied ? (because this command is new). Any reason for this ?

Reply:
Because we auto-generated the certs inside the component itself, it automatically put it into the right place. Handing that off to OLM is new in this PR.